tests/unittests/config/test_cc_grub_dpkg.py

```diff
@@ -247,7 +247,7 @@ def test_handle(
         """Test setting of correct debconf database entries"""
         m_is_efi_booted.return_value = is_uefi
         m_fetch_idevs.return_value = fetch_idevs_output
-        cfg = {"grub_dpkg": {}}
+        cfg = {"grub_dpkg": {"enabled": True}}
```

Comment:
It appears that non-cloud livecd-rootfs images still rely on this cloud-init grub boot traget setup per-boot. This behavior should not land yet until we have sorted expected behavior in those environments.

Reply:
> It appears that non-cloud livecd-rootfs images still rely on this cloud-init grub boot traget setup per-boot

Those are images, and therefore should be able to enable this as needed in `cloud.cfg.d/`.